configs/metadata.yaml

```diff
@@ -39,51 +39,86 @@ s2:
       swir16: 450
       swir22: 480
     wavelength:
-      blue: 0.4905
-      green: 0.5605
+      blue: 0.49
+      green: 0.56
       red: 0.665
-      rededge1: 0.7055
-      rededge2: 0.7405
+      rededge1: 0.704
+      rededge2: 0.74
       rededge3: 0.783
-      nir: 0.8425
+      nir: 0.842
       nir08: 0.865
       swir16: 1.61
       swir22: 2.19
-l8:
+landsat-c2l1:
   band_order:
-    - blue
+    - red
     - green
+    - blue
+    - nir08
+    - swir16
+    - swir22
```

Comment:
Why is the near-infrared band for Landsat called nir08? Band 8 on Landsat 8/9 is the panchromatic band. Is this to match with Sentinel-2? Maybe we could follow the HLS naming scheme at https://lpdaac.usgs.gov/data/get-started-data/collection-overview/missions/harmonized-landsat-sentinel-2-hls-overview/#hls-spectral-bands if we want to keep the same band name. I.e. use NIR Broad, NIR Narrow, SWIR 1, SWIR 2.

Reply:
The naming comes as-is from the underlying STAC catalog. We have this baked into the code in 2-3 places now. So let's not change it right now and make this more consistent in a future iteration.

Example:

https://landsatlook.usgs.gov/stac-server/collections/landsat-c2l1/items/LC09_L1GT_173239_20240430_20240430_02_T2

Reply:
Ah I see, then let's keep this as is for now then.